src/Form/Login.php

```diff
@@ -27,7 +27,7 @@ class Login extends Form
     public $cookieWarning = 'This website uses web cookie to remember you while you are logged in.';
 
     /**
-     * Intialization.
+     * Initialization.
```

Comment:
just no comment ;-)

Reply:
hahaha you are right, I just look around in mode: "Berserk, fix typos WHATEVER IT TAKES"
I remove the obvious :D